HinglishULMFiT_AWD_QRNN.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>How are the params selected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<ul><li>Learning rate - Based on suggestion from <code>lr_find()</code></li><li>Epochs - randomly based on which ones gave better results. </li></ul><p> </p>

Reply:
Add results of `lr_find` in the notebook somewhere?